src/ophyd_async/epics/adcore/_core_io.py

```diff
@@ -70,14 +70,14 @@ class NDPluginStatsIO(NDPluginBaseIO):
     # Basic statistics
     compute_statistics: A[SignalRW[bool], PvSuffix.rbv("ComputeStatistics")]
     bgd_width: A[SignalRW[int], PvSuffix.rbv("BgdWidth")]
-    total_array: A[SignalRW[float], PvSuffix.rbv("TotalArray")]
+    total_array: A[SignalRW[float], PvSuffix("TotalArray", "Total_RBV")]
     # Centroid statistics
     compute_centroid: A[SignalRW[bool], PvSuffix.rbv("ComputeCentroid")]
     centroid_threshold: A[SignalRW[float], PvSuffix.rbv("CentroidThreshold")]
     # X and Y Profiles
     compute_profiles: A[SignalRW[bool], PvSuffix.rbv("ComputeProfiles")]
-    profile_size_x: A[SignalRW[int], PvSuffix.rbv("ProfileSizeX")]
-    profile_size_y: A[SignalRW[int], PvSuffix.rbv("ProfileSizeY")]
+    profile_size_x: A[SignalR[int], PvSuffix("ProfileSizeX")]
+    profile_size_y: A[SignalR[int], PvSuffix("ProfileSizeY")]
```

Comment:
According to the amended [docs](https://areadetector.github.io/areaDetector/ADCore/NDPluginStats.html), shouldn't these be `_RBV`?
```suggestion
    profile_size_x: A[SignalR[int], PvSuffix("ProfileSizeX_RBV")]
    profile_size_y: A[SignalR[int], PvSuffix("ProfileSizeY_RBV")]
```

Reply:
Good catch, removed the wrong one

Reply:
I think (but am checking) that could also 
```suggestion
    profile_size_x: A[SignalR[int], PvSuffix.rbv("ProfileSizeX")]
    profile_size_y: A[SignalR[int], PvSuffix.rbv("ProfileSizeY")]
```
But I am not sure if having both prefixes is allowed when making a SignalR only